test/fixtures/config.json

```diff
@@ -38,5 +38,26 @@
       "S7",
       "S8"
     ]
+  },
+  "devices": {
```

Comment:
there might be different type of devices, like Denon, zWave, etc..  all with different required parameters.  Maybe for now just have 'devices_zwave', 'devices_roku, etc?